src/lib/search-and-replace.js

```diff
@@ -58,7 +52,7 @@ export function getReadAndWriteStreams( {
 	let outputFileName;
 
 	if ( inPlace ) {
-		const midputFileName = path.join( makeTempDir(), path.basename( fileName ) );
+		const midputFileName = path.join( makeTempDir( 'vip-search-replace' ), path.basename( fileName ) );
```

Comment:
Could we please clean up the temporary directory when we are done?

Reply:
Yes, we clean up the files, but I actually was talking about the temporary directory itself :-) Say, when I run tests locally, I end up with many temporary directories created by search/replace.

I was thinking on smth like this (not tested; probably there is a better alternative):

```js
export function makeTempDir( prefix = 'vip-cli' ) {
    const tempDir = fs.mkdtempSync( path.join( os.tmpdir(), `${ prefix }-` ) );
    debug( `Created a directory to hold temporary files: ${ tempDir }` );

    process.on( 'exit', () => fs.rmSync( tempDir, { recursive: true } );
    return tempDir;
}
```